packages/account/src/providers/transaction-summary/input.ts

```diff
@@ -37,10 +37,7 @@ export function getInputFromAssetId(inputs: Input[], assetId: string) {
   const coinInputs = getInputsCoin(inputs);
   const messageInputs = getInputsMessage(inputs);
   const coinInput = coinInputs.find((i) => i.assetId === assetId);
-  // TODO: should include assetId in InputMessage as well. for now we're mocking ETH
-  const messageInput = messageInputs.find(
-    (_) => assetId === '0x0000000000000000000000000000000000000000000000000000000000000000'
-  );
+  const messageInput = messageInputs.find(({ amount }) => !!amount && amount.gt(0));
```

Comment:
Do we not need to validate the base asset ID here?

Reply:
[`InputMessage`](https://docs.fuel.network/docs/specs/tx-format/input/#inputmessage) doesn't have an asset id to validate against.

Reply:
I think this is because it will always be for the base asset ID, as the message is used for withdrawals in the base asset i.e. bridging. That check did work in the past when the base asset was hard coded.

Reply:
It turns out the the [ReceiptCall's](https://docs.fuel.network/docs/specs/abi/receipts/#call-receipt)  `baseAssetId`  is always `0x0000000000000000000000000000000000000000000000000000000000000000` but this doesn't provide the correct `InputMessage` for predicate / contract calls **AND** also for withdrawals, therefore I opted to use the `amount` field as a delineating factor.